docs/rst/tutorials/core/qos/xml_profiles/xml_profiles.rst

```diff
@@ -0,0 +1,400 @@
+  The background thread is in charge of consuming the queue and sending the data to every matched reader.
+* **SYNCHRONOUS**: synchronous publication mode.
+  Setting this mode implies that the data is sent directly within the context of the user thread.
+  This entails that any blocking call occurring during the write operation would block the user thread,
+  thus preventing the application from continuing its operation.
+  It is important to note that this mode typically yields higher throughput rates at lower latencies,
+  since there is no notification nor context switching between threads.
+* **AUTO**: let Fast DDS select the publication mode.
+  This implies using the publication mode set in the :ref:`XML file <vulcanexus_configure_xml>`, or otherwise,
+  the default value set in Fast DDS (see |PublishModeQosPolicy|).
+
+*rmw_fastrtps* defines two configurable parameters in addition to
+`ROS 2 QoS policies <https://index.ros.org/doc/ros2/Concepts/About-Quality-of-Service-Settings/#qos-policies>`_.
+Said parameters, and their default values under ROS 2, are:
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - Parameter
+     - Description
+     - Default ROS 2 value
+   * - |MemoryManagementPolicy|
+     - *Fast DDS* preallocates memory for the publisher |br|
+       and subscriber histories. When those histories fill |br|
+       up, a reallocation occurs to reserve more memory.
+     - |PREALLOCATED_WITH_REALLOC_MEMORY_MODE-api|
+   * - |PublishModeQosPolicy|
+     - User calls to publication method add the messages |br|
+       in a queue that is managed in a different thread, |br|
+       meaning that the user thread is available right |br|
+       after the call to send data.
+     - |ASYNCHRONOUS_PUBLISH_MODE-api|
+
+
+.. _vulcanexus_configure_xml:
+
+XML configuration
+-----------------
+
+To use specific *Fast-DDS* features within a ROS 2 application,
+XML configuration files can be used to configure a wide set of *QoS*.
+Please refer to |XML_profiles| for Fast DDS native applications in Fast DDS's documentation to see the whole list of configuration options available in *Fast DDS*.
+
+When configuring *rmw_fastrtps* using XML files, there are certain points that have to be taken into account:
+
+* ROS 2 QoS contained in `rmw_qos_profile_t <http://docs.ros2.org/latest/api/rmw/structrmw__qos__profile__t.html>`_
+  are always honored, unless set to ``*_SYSTEM_DEFAULT``.
+  In that case, XML values, or Fast DDS default values in the absences of XML ones, are applied.
+  This means that if any QoS in ``rmw_qos_profile_t`` is set to something other than ``*_SYSTEM_DEFAULT``,
+  the corresponding value in the XML is ignored.
+
+* By default, *rmw_fastrtps* overrides the values for |MemoryManagementPolicy| and |PublishModeQosPolicy|.
+  This means that the values configured in the XML for these two parameters will be ignored.
+  Instead, |PREALLOCATED_WITH_REALLOC_MEMORY_MODE-api| and |ASYNCHRONOUS_PUBLISH_MODE-api|
```

Comment:
In general, we don't have a max line length in this project, so do not break lines unles `.`